spec/requests/storage_services_spec.rb

```diff
@@ -30,4 +30,49 @@
       expect(response.parsed_body).to include(expected)
     end
   end
+
+  context "POST /api/host_initiators" do
+    it "creates new Storage Service" do
+      api_basic_authorize(collection_action_identifier(:storage_services, :create))
+      provider = FactoryBot.create(:ems_autosde, :name => 'Autosde')
```

Comment:
Unless the name is required best to leave it off
```suggestion
      provider = FactoryBot.create(:ems_autosde)
```